cmd/inspect.go

```diff
@@ -56,13 +76,224 @@ var (
 			return index_usage.Run(ctx, dbConfig, fsys)
 		},
 	}
+
+	inspectLocksCmd = &cobra.Command{
+		Use:   "locks",
+		Short: "Shows queries which have taken out an exclusive lock on a relation",
```

Comment:
```suggestion
		Short: "Show queries which have taken out an exclusive lock on a relation",
```

nitpick: let's drop the `s` for consistency with other commands. see `supabase help`